requirements.txt

```diff
@@ -12,3 +12,9 @@ Flask-SQLAlchemy==2.3.2
 colorama==0.3.9
 colour==0.1.5
 psycopg2-binary==2.7.5
+webargs==4.0.0
+Werkzeug==0.14.1
```

Comment:
AFAIK Werkzeug is a requirement of flask. I don't think that we need to add here.

Reply:
I have this error without Werkzeug:
```
Traceback (most recent call last):
  File "examples/dh.py", line 4, in <module>
    from teal.teal import Teal
  File "/home/ribaguifi/ereuse-tag/tag/env/lib/python3.7/site-packages/teal/teal.py", line 15, in <module>
    from werkzeug.wsgi import DispatcherMiddleware
ImportError: cannot import name 'DispatcherMiddleware' from 'werkzeug.wsgi' (/home/ribaguifi/ereuse-tag/tag/env/lib/python3.7/site-packages/werkzeug/wsgi.py)
```

Reply:
Ok! My fault, seems that Werkzeug is the dependency problem to fix #4.

Reply:
Ok, Now is done.
test requirements is in requirements-test.txt and alembic is out.